### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by hemantdwave**
Add integration test

### CODE REVIEW

File: tests/test_integration.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# limitations under the License.
+
+import subprocess
+import unittest
+import os
+
+class IntegrationTests(unittest.TestCase):
+
+    def test_antenna_selection(self):
+        output = subprocess.check_output(["python", "antennas.py"])
```
Comment (m3ller) [inline tests/test_integration.py]: rather than "python", please `import sys` and use `sys.executable`.

Comment (m3ller) [inline tests/test_integration.py]: This is test is also an improvement to the existing `test_antennas.py` code. Could you simply make your changes in `test_antennas.py` as both files simply do a smoke test (yours is a more in-depth version!).

Comment (hemantdwave) [reply tests/test_integration.py]: @m3ller my plan is to add more integration tests , so created separate file

File: tests/test_antennas.py [hunk 2]
```diff
@@ -27,3 +27,19 @@ def test_smoke(self):
...
         subprocess.check_output([sys.executable, demo_file])
+
+    def test_antenna_selection(self):
+        """ Verify contents of output """
+        
+        demo_file = os.path.join(project_dir, 'antennas.py')
+        output = subprocess.check_output([sys.executable, demo_file])
+        output = str(output).upper()
+        if os.getenv('DEBUG_OUTPUT'):
+            print("Example output \n"+ output)
```
Comment (randomir) [inline tests/test_antennas.py]: whitespace

**Review by randomir — APPROVED** [review]

**Pull request merged**